apps/amo/tests/test_views.py

```diff
@@ -390,3 +390,13 @@ def test_contribute_json(self):
         res = self.client.get('/contribute.json')
         eq_(res.status_code, 200)
         eq_(res._headers['content-type'], ('Content-Type', 'application/json'))
+
+
+class TestRobots(amo.tests.TestCase):
+
+    def test_disable_collections(self):
+        """Make sure /en-US/firefox/collections/ gets disabled"""
+        url = reverse('collections.list')
+        response = self.client.get('/robots.txt')
+        eq_(response.status_code, 200)
```

Comment:
Yeah, what I've used till now is whenever I write a new test, or modify an existing one, I:
1/ use `assert`s instead of `eq_`, `ok_` and `self.assert...`
2/ get rid of the one letter variables that plague the code

We could indeed get rid of nose altogether at some point, I guess.